docs/docs/config/system.md

```diff
@@ -29,10 +29,11 @@ Making changes to any of the settings in this section modifies the HID report de
 
 :::
 
-| Config                                | Type | Description                                                    | Default |
-| ------------------------------------- | ---- | -------------------------------------------------------------- | ------- |
-| `CONFIG_ZMK_HID_INDICATORS`           | bool | Enable receipt of HID/LED indicator state from connected hosts | n       |
-| `CONFIG_ZMK_HID_CONSUMER_REPORT_SIZE` | int  | Number of consumer keys simultaneously reportable              | 6       |
+| Config                                       | Type | Description                                                    | Default |
+| -------------------------------------------- | ---- | -------------------------------------------------------------- | ------- |
+| `CONFIG_ZMK_HID_INDICATORS`                  | bool | Enable receipt of HID/LED indicator state from connected hosts | n       |
+| `CONFIG_ZMK_HID_CONSUMER_REPORT_SIZE`        | int  | Number of consumer keys simultaneously reportable              | 6       |
+| `CONFIG_ZMK_HID_SEPARATE_MOD_RELEASE_REPORT` | bool | Release the Modifiers separate from and after the modified key | n       |
```

Comment:
IMHO, this description doesn't make sense to me. Perhaps "Release modifiers before the key they modify" or somesuch? Maybe @caksoylar has a better idea for a succinct description of this.

Reply:
"Send modifier key release events before non-modifiers"? Given this is practically happening only for modifier functions, we could add "for modifier functions" perhaps.

Reply:
But isn't it the other way around?
We first send the normal key release event and then the modifier release after that.

Reply:
May be something like: "Send Modifier release event after Non-Modifier release event."

Reply:
> But isn't it the other way around?
We first send the normal key release event and then the modifier release after that.

Yes, I mixed it up in my comment. I also wrote the "before" version rather than "after" since technically the new report is the earlier one, but functionally either way works.

Reply:
Ok perfect. Then my commit should resolve the issue. :)